website/src/data/users.js

```diff
@@ -314,6 +314,14 @@ const Users = [
     source: 'https://github.com/MisterFISHUP/covid-19-in-france',
     tags: ['opensource', 'i18n', 'large'],
   },
+  {
+    title: 'Dart Code Metrics',
+    description: 'Static analysis tool that helps analyse and improve Dart code quality.',
+    preview: require('./showcase/dart-code-metrics.png'),
+    website: 'https://dartcodemetrics.dev/',
```

Comment:
Just wanted to understand the reason behind adding a forward slash in the url

Reply:
Copy pasted the url from the browser url bar. Is it wrong to have it?

Reply:
No just wanted to know, the reason. Both url works the same way, if URLs are being used as a Baseurl in the code somewhere, it depends if they want to join the URI fragments with a slash or without.